frontend/src/components/worksheets/Worksheet.js

```diff
@@ -806,40 +812,67 @@ class Worksheet extends React.Component {
 
         var searchClassName = this.state.showActionBar ? '': 'search-hidden';
         var editableClassName = canEdit ? 'editable' : '';
-        var viewClass = !canEdit && !this.state.editMode ? 'active' : '';
-        var rawClass = this.state.editMode ? 'active' : '';
         var disableWorksheetEditing = this.canEdit() ? '' : 'disabled';
-        var sourceStr = editPermission ? 'Edit Source' : 'View Source';
+        var sourceStr = editPermission ? 'Edit' : 'View';
```

Comment:
I think "Edit Source" is clearer because it makes it clear that we're actually editing the worksheet source.

Can you add screenshots to this PR of all new screens? (for example, how the "View" button looks like when you're actually editing the source)